README.md

```diff
@@ -35,4 +31,224 @@ In order to test changes to helm charts, you can run the following script:
 ```
 ./startLocalTest.sh
 ```
-This will package all the helm charts from the local directories, and launch the cluster.
\ No newline at end of file
+This will package all the helm charts from the local directories, and launch the cluster.
+
+## DataWave Configuration and Deployment
+
+### Required Information for onboarding a new datatype
+- name of the datatype
+- What fields will be queryable
+  - types for the fields
+  - what fields are unique identifiers
+
+### Steps for Adding a new Datatype
+1) Open the datawave-stack/values.yaml or your own values.yaml.
```

Comment:
Perhaps just talk about what should be done to add to your own values override file. Most people should be just using the stack charts, once we get to a point of deploying them, so this should tell a user how to override/add their own